src/v2/Apps/Order/Routes/Review/index.tsx

```diff
@@ -133,7 +133,27 @@ export class ReviewRoute extends Component<ReviewProps> {
           o => o.lineItems?.edges?.[0]?.node?.artwork?.slug
         )
 
-        // Eigen redirects to the status page (must keep the user inside the webview)
+        if (
+          order.mode === "OFFER" &&
+          isEigen &&
+          order.source === "artwork_page"
+        ) {
```

Comment:
#superminor - since this is very specifically Eigen related, it might read slightly better and exit early if that condition was first:
```tsx
isEigen &&
order.mode === "OFFER" &&
order.source === "artwork_page"
```